src/engraving/accessibility/accessibleitem.cpp

```diff
@@ -25,14 +25,34 @@
 #include "../libmscore/score.h"
 #include "../libmscore/measure.h"
 
+#include "translation.h"
 #include "log.h"
 
+using namespace mu;
 using namespace mu::engraving;
 using namespace mu::accessibility;
 using namespace mu::engraving;
 
 bool AccessibleItem::enabled = true;
 
+static QString readable(QString s)
+{
+    // Remove undesired pauses in screen reader speech output
+    s.remove(u':');
+
+    // Handle desired pauses
+    s.replace(u';', u','); // NVDA doesn't pause for semicolon
+
+    // Replace common Unicode characters that screen readers can't pronounce
```

Comment:
Hm, there is my fix of this bug https://github.com/musescore/MuseScore/pull/12308/commits/5e0a8460564e6ce2ecc730faa5e5f93d39453cc4

Reply:
Ok, it seems that this really does just occur with Narrator so we can probably consider the non-spoken accidentals Microsoft's bug rather than our own. Your version of the fix will still be useful though. Firstly for Narrator, but also to VoiceOver because it says "music flat sign" instead of just "flat", etc, which gets annoying quite quickly.